src/VisualStudio/Xaml/Impl/XamlStaticTypeDefinitions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml
+{
+    public static class XamlStaticTypeDefinitions
```

Comment:
@jasonmalinowski may want to chime in here.  The editor synthesizes a content type for you based on your language service registration.  I wouldn't expect you to need this (except to get the `BaseDefinition` in there.  In either case I would expect to need the extension mapping here.